app/routes.py

```diff
@@ -1,2 +1,39 @@
-from flask import Blueprint
+# localhost:5000/   <-- add url endpoint/parameters here
 
+from flask import Blueprint, jsonify
+
+
+class Planet():
+    def __init__(self, id, name, description, xenomorphs=False):
+        self.id = id
+        self.name = name
+        self.description = description
+        self.xenomorphs = xenomorphs
+
+
+PLANETS = [
+
+    Planet(426, "Nostromo's End", "Hostile weather.  Toxic atmosphere.  Evidence of civilization.", True),
+    Planet(224, "JollyPlanet", "Okay. Decent. Will live for long time.", True)
+
+]
+
+planets_bp = Blueprint("planets_bp", __name__, url_prefix="/planets")
+
```

Comment:
Minor note, consider moving extra spaces:
```suggestion
from flask import Blueprint, jsonify

class Planet():
    def __init__(self, id, name, description, xenomorphs=False):
        self.id = id
        self.name = name
        self.description = description
        self.xenomorphs = xenomorphs

PLANETS = [
    Planet(426, "Nostromo's End", "Hostile weather.  Toxic atmosphere.  Evidence of civilization.", True),
    Planet(224, "JollyPlanet", "Okay. Decent. Will live for long time.", True)
]

planets_bp = Blueprint("planets_bp", __name__, url_prefix="/planets")

```